src/benchmarks/gc/src/commonlib/setup.py

```diff
@@ -11,15 +11,19 @@
 
 from .command import Command, CommandKind, CommandsMapping
 from .config import PERFVIEW_PATH, SIGCHECK64_PATH
+from .get_built import is_arm
 from .host_info import write_host_info
 from .util import assert_file_exists, ensure_dir, os_is_windows, unlink_if_exists
 
 
 def setup() -> None:
     if os_is_windows():
         _download_perfview_and_sigcheck()
-    # Need sigcheck to do this
-    write_host_info()
+
+    # Need sigcheck to do this. Also, since the host info tools are not
+    # available on ARM, skip when on said architecture.
+    if not is_arm():
+        write_host_info()
```

Comment:
I imagine people (like me :p) would forget doing the manual host info specification.
What would happen if the host info were not manually specified?
Can we emit a warning somewhere?

Reply:
That's an interesting observation worth adding. Will work on it.